db/models.py

```diff
@@ -13,4 +13,40 @@ class Actor(models.Model):
     last_name = models.CharField(max_length=255)
 
     def __str__(self) -> str:
-        return f"{self.first_name} {self.last_name}"
+        return self.first_name + " " + self.last_name
```

Comment:
It's better to use f-string formatting for string concatenation. It's more readable and faster. So, it would be better if you rewrite this line as: `return f'{self.first_name} {self.last_name}'`